src/main/java/uk/co/nazire/controller/GlobalExceptionHandling.java

```diff
@@ -0,0 +1,35 @@
+package uk.co.nazire.controller;
+
+import java.util.Date;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+
+import uk.co.nazire.exception.DataAlreadyExistException;
+import uk.co.nazire.exception.DataNotFoundException;
+import uk.co.nazire.model.ErrorDetails;
+
+@RestControllerAdvice
+public class GlobalExceptionHandling {
+	
+	@ExceptionHandler(DataNotFoundException.class)
+	@ResponseStatus(HttpStatus.NOT_FOUND)
+	public ErrorDetails dataNotFoundExceptionHandler(final DataNotFoundException ex,final WebRequest request) {
+		
+		ErrorDetails errorDetails = new ErrorDetails(ex.getMessage(),request.getDescription(false),
```

Comment:
You should to use space of the parameters.

Reply:
I have done. I should be carefully after that.